testing/tests/ui/rinja-block.stderr

```diff
@@ -7,10 +7,10 @@ error: unknown node `fail`
   | ^^^^^^^^^^^^^^^^^^^^^^
 
 error: when using `in_doc = true`, the struct's documentation needs a `rinja` code block
-  --> tests/ui/rinja-block.rs:15:25
+  --> tests/ui/rinja-block.rs:15:34
    |
 15 | #[template(ext = "txt", in_doc = true)]
```

Comment:
Shouldn't it underline the whole `in_doc = true`?

Reply:
In the other cases we also only underline the value part, the part after the equals sign. I actually came to like that, because the problem is with the value, not with the key. If you write `in_doc = false`, then everything is fine (except that in here there is not source).

Reply:
Hmmmmmmm. Then the message needs to be reworded a bit to make it obvious that the value is the value. Like:

> when using `in_doc` with the `true` value, the struct's documentation needs a `rinja` code block

Or something like that.

Reply:
I changed the message a bit. Do you think it's more clear now?

Reply:
Indeed, thanks!